src/pages/DataAccessRequestApplication.js

```diff
@@ -1612,44 +1612,24 @@ class DataAccessRequestApplication extends Component {
                     div({ className: 'row no-margin' }, [
                       div({ className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group' }, [
                         label({ className: 'control-label default-color' },
-                          ['1. Download the Data Access Agreement template and have your organization\'s Signing Official sign it'])
+                          ['By submitting this data access request, you agree to comply with all terms relevant to Authorized Users put forth in the agreement.'])
                       ]),
 
                       div({ className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group' }, [
                         a({
-                          id: 'link_downloadAgreement', href: 'BroadDataAccessAgreement.pdf', target: '_blank',
+                          id: 'link_downloadAgreement', href: 'DUOS_LibraryCardAgreementTemplate_3.28.20_PIaacknowledgement.pdf', target: '_blank',
```

Comment:
Typo ... should be `DUOS_LibraryCardAgreementTemplate_3.28.20_PIacknowledgement.pdf`